.github/workflows/ff-release-candidate-to-master.yml

```diff
@@ -8,7 +8,60 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  fast-forward:
+  validate-version:
```

Comment:
fails `ff-rc-to-master` workflow if the version was not bumped

<img width="775" alt="Screenshot 2025-03-06 at 18 39 47" src="https://github.com/user-attachments/assets/85782054-b580-42e4-aa17-2e761c0df2c5" />